3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -16,28 +16,54 @@
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
 function requestData(url) {
+  //This works fine without checking HTTP Errors
   // TODO return a promise using `fetch()`
+  return fetch(url).then((response) => response.json());
 }
 
+//This returns "Ooooops! Something went wrong!:Cannot read property 'img' of undefined"
+// function requestData(url) {
+//   // TODO return a promise using `fetch()`
+//   fetch(url).then((response) => {
```

Comment:
You forgot the `return` keyword to return promise from `fetch()`:

```suggestion
//   return fetch(url).then((response) => {
```